docs/design.md

```diff
@@ -1,5 +1,11 @@
 # Presidio design
 
-## Table of contents
+## Analyzer
+[![Analyzer Design](assets/analyzer-design.png)](analyzer/index.md)
```

Comment:
@omri374 Should I use absolute path here as well? How is the linking handled in mkdocs?